phpBB/adm/style/acp_ext_list.html

```diff
@@ -52,6 +52,7 @@
 			<td style="text-align: center;">
 				<!-- IF enabled.S_VERSIONCHECK -->
 				<strong <!-- IF enabled.S_UP_TO_DATE -->style="color: #228822;"<!-- ELSE -->style="color: #BC2A4D;"<!-- ENDIF -->>{enabled.META_VERSION}</strong>
```

Comment:
Can we also do something about these inline styles? I would prefer to see a class used here, the same class used to style the icon color, so maybe just rename the class `outdated-ext` and use it for the text and the icon.

Reply:
Done ;)